sigs.yaml

```diff
@@ -527,12 +527,10 @@ sigs:
       chairs:
       - name: Justin Santa Barbara
         github: justinsb
-      - name: Kris Nova
-        github: kris-nova
-        company: VMware
       - name: Nishi Davidson
         github: d-nishi
-        company: AWS
+      - name: Kris Nova
+        github: kris-nova
```

Comment:
Should Kris' affiliation be maintained?

Reply:
Also... could you move this change to a separate commit since it's unrelated to adding subprojects? :grimacing:

Reply:
this was wrong as Kris was linked to Heptio. I made it consistent with other co-chairs.

I have it in the commit history that the PRs make 2 changes. Its such a small change - do I really need to separate them?